plugins/console.js

```diff
@@ -23,7 +23,7 @@ var logForGivenLevel = function(level) {
     if (level === 'warn') level = 'warning';
     return function () {
         var args = [].slice.call(arguments);
-        Raven.captureMessage('' + args, {level: level, logger: 'console'});
+        Raven.captureMessage('' + args[0], {level: level, logger: 'console', extra: { args: args }});
```

Comment:
Would we want to do `{ args: args.slice(1) }` instead? Since this is already sending the first argument as the message.

Also, I think I'd prefer to send the key as `arguments` rather than `args` since `arguments` is what it's really called. :)